client/src/api.ts

```diff
@@ -25,35 +46,144 @@ export const createElection = async (data: {
       electionId: `uuid-${Date.now()}`, // This will be a UUID
     }
   }
-  return await fetchJSON('/elections', {
+  return await fetchJSON(`/api/mailelection`, {
     method: 'post',
     body: JSON.stringify(data),
   })
 }
 
-export const getElections = async () => {
+export const getElections = async (): Promise<{
+  mailElections: MailElection[]
+}> => {
   if (useFakeAPI) {
     await sleep()
-    return {
-      elections: [
-        {
-          electionId: 'asdf',
-        },
-      ],
-    }
+    return { mailElections: [mailElection] }
   }
-  return await fetchJSON('/elections')
+  return await fetchJSON(`/api/mailelection`)
 }
 
 export const getElection = async (
   electionId: string
-): Promise<ElectionsListItem> => {
+): Promise<MailElection> => {
   if (useFakeAPI) {
-    return {
-      id: electionId,
-      name: 'foo',
-      createdAt: new Date().toISOString(),
-    }
+    return mailElection
   }
-  return fetchJSON(`/elections/${electionId}`)
+  return fetchJSON(`/api/mailelection/${electionId}`)
 }
+
+export const approveElection = async (data: {
+  electionId: string
+  userId: string
```

Comment:
This `userId` is implicit in the request, right?

Reply:
Yes. Good point.